script/tool/lib/src/native_test_command.dart

```diff
@@ -367,10 +367,21 @@ this command.
               'notAnnotation=io.flutter.plugins.DartIntegrationTest';
 
           print('Running integration tests...');
+          // Explicitly request all ABIs, as Flutter would if being called
+          // without a specific target (see
+          // https://github.com/flutter/flutter/pull/154476) to ensure it can
+          // run on any architecture emulator.
+          const List<String> abis = <String>[
```

Comment:
risc-v will eventually need to be added to this list but we can add it when we add risc-v testing.